### PR TITLE
docs: fill doc string of `VariablePassingValue` and `VariablePassingStructValue`

### PR DESCRIPTION
**Pull request by rokamu623**
## Pre-review checklist for the PR author

- [ ] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
- [ ] The PR is ready for review.

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] (Optional) The PR has been properly tested with CARET_report verification.
- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

File: src/caret_analyze/value_objects/variable_passing.py [hunk 1]
```diff
@@ -40,7 +55,7 @@ def node_name(self) -> str:
         Returns
         -------
         str
-            node name
+            Node name
```
Comment (emori-ctc) [inline src/caret_analyze/value_objects/variable_passing.py]: Please append a period.

File: src/caret_analyze/value_objects/variable_passing.py [hunk 2]
```diff
@@ -93,7 +121,7 @@ def node_name(self) -> str:
...
```
Comment (emori-ctc) [inline src/caret_analyze/value_objects/variable_passing.py]: Please append a period.

**Review by emori-ctc — CHANGES_REQUESTED** [review]

File: src/caret_analyze/value_objects/variable_passing.py (hunk 1 above)
Comment (rokamu623) [reply src/caret_analyze/value_objects/variable_passing.py]: Fixed it in [053d3c9](https://github.com/rokamu623/CARET_analyze/pull/23/commits/053d3c9bb8f0879da4c23ed92c13a96ef6728faf).

File: src/caret_analyze/value_objects/variable_passing.py (hunk 2 above)
Comment (rokamu623) [reply src/caret_analyze/value_objects/variable_passing.py]: Fixed it in [053d3c9](https://github.com/rokamu623/CARET_analyze/pull/23/commits/053d3c9bb8f0879da4c23ed92c13a96ef6728faf).

**Review by emori-ctc — APPROVED** [review]
LGTM!

**Pull request merged**